config.py

```diff
@@ -57,7 +57,14 @@ class Config(object):
     ACCOUNT_EMAIL = os.getenv("ACCOUNT_EMAIL")
     STUDENT_ORG_UNIT = os.getenv("STUDENT_ORG_UNIT")
     SCHOOL_YEAR_START = os.getenv("SCHOOL_YEAR_START")
-    DB_TYPE = os.getenv("DB_TYPE")
+    DB_DIALECT = os.getenv("DB_DIALECT")
+    DB_USER = os.getenv("DB_USER")
+    DB_PWD = os.getenv("DB_PWD")
+    DB_SERVER = os.getenv("DB_SERVER")
+    DB_PORT = os.getenv("DB_PORT")
+    DB = os.getenv("DB")
+    DB_SCHEMA = os.getenv("DB_SCHEMA")
+    DB_CREDS = os.getenv("DB_CREDS")
```

Comment:
I generally like to err on the side of clarity when there's any question of what a variable might be — can we rename this to `DB_CREDENTIALS`?